Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -373,11 +481,13 @@ void MaxEnt::exec() {
     Progress progress(this, 0.0, 1.0, nIter);
 
     // Run maxent algorithm
+    bool stop = false;
```

Comment:
I dont think stop is the best name, I would assume it has more to do with a user cancelling the algorithm than it having converged before the maximum number of iterations.